Update install-python-compatibility.md (#5012)
fishtown-analytics/docs.getdbt.com: website/docs/faqs/Core/install-python-compatibility.md

id: install-python-compatibility
---

> You can use the followign table to match your dbt-core version with the compatible version of Python. New dbt minor versions will add support for new Python3 minor versions as soon as all dependencies can support it. In addition, dbt minor versions will drop support for old Python3 minor versions right before they reach [end of life](https://endoflife.date/python).

Author: ```suggestion
Use this table to match dbt-core versions with their compatible Python versions. New dbt minor versions will add support for new Python3 minor versions as soon as all dependencies can support it. In addition, dbt minor versions will drop support for old Python3 minor versions right before they reach [end of life](https://endoflife.date/python).
```

> | dbt-core → <br/> Python ↓ | 1.0 | 1.1 | 1.2 | 1.3 | 1.4 | 1.5 | 1.6 | 1.7 | 1.8 |

Author: @matthewshaver @mirnawong1 and @nghi-ly I wonder if we should switch tables with ✅ and ❌  to this greyed out x? It's so much easier to read. You focus on the ✔️ instead of everything

Reviewer: @runleonarun : my 2 cents, the page feels a little bit "washed out" with the ✔️
